airflow/migrations/versions/c381b21cb7e4_add_session_table_to_db.py

```diff
@@ -0,0 +1,60 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""add session table to db
+
+Revision ID: c381b21cb7e4
+Revises: e655c0453f75
+Create Date: 2022-01-25 13:56:35.069429
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.engine.reflection import Inspector
+
+# revision identifiers, used by Alembic.
+revision = 'c381b21cb7e4'
+down_revision = 'e655c0453f75'
+branch_labels = None
+depends_on = None
+
+TABLE_NAME = 'sessions'
+
+
+def upgrade():
+    """Apply add session table to db"""
+    conn = op.get_bind()
+    inspector = Inspector.from_engine(conn)
```

Comment:
We are trying to implement running migrations offline starting from 2.0.0 migration head and this would be a blocker.
PR: https://github.com/apache/airflow/pull/20962

Reply:
Interesting, thanks for pointing me to that. I'll remove that then and have the upgrade assume that the table doesn't exist.

Reply:
Marking this as resolved since I believe I fixed the issue. Please review again and let me know if I need to unresolve it.